src/dragDrop.js

```diff
@@ -1,38 +1,43 @@
-let current = null;
-let targetItem = null;
+let target;
 
-export function dragStart() {
-  current = this;
-  current.classList.add('current-active');
-}
-
-export function dragEnd() {
-  current.classList.remove('current-active');
-  current = null;
-}
+function sorting(source, target) {
+  const savedList = JSON.parse(localStorage.getItem('savedList'));
+  if (savedList.length < 2) return;
 
-export function dragEnter(event) {
-  event.preventDefault();
-}
+  const sourceObj = savedList[source];
+  const souceIndex = savedList[source].index;
+  let targetIndex;
+  savedList.forEach((obj) => {
+    if (obj.index === Number(target)) {
+      targetIndex = savedList.indexOf(obj);
+    }
+  });
 
-export function dragLeave() {
-  targetItem = null;
-}
+  savedList[source].index = savedList[targetIndex].index;
+  savedList[targetIndex].index = souceIndex;
 
-export function allowDrop(event) {
-  event.preventDefault();
+  savedList[source] = savedList[targetIndex];
+  savedList[targetIndex] = sourceObj;
+  localStorage.setItem('savedList', JSON.stringify(savedList));
 }
 
-export function drop(event) {
-  event.preventDefault();
-  targetItem = document.getElementById(event.target.id);
-  current.parentElement.insertBefore(current, targetItem);
-
-  const children = Array.from(current.parentElement.children);
-  const updatedList = children.map((el, index) => ({
-    index,
-    completed: el.checked,
-    description: el.children[1].value,
-  }));
-  localStorage.setItem('TodoList', JSON.stringify(updatedList));
+export default function dragAndDrop(event, index) {
+  const newEvent = event.type;
```

Comment:
- [ ] I would recommend using ES6 syntax. Use arrow function instead of `function` keyword to make the code cleaner.